recipes/radiospectra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "radiospectra" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "94840b28d38b6a811bec8e601a9bcf9d789e44fbeb3ebf173d303fcb38e7864a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
```

Comment:
Can this use pip? https://conda-forge.org/docs/meta.html#use-pip

Reply:
No, I don't think it can... (or at least I don't know how and with pip it failed). This package uses a submodule ([astropy-helpers](https://github.com/astropy/astropy-helpers)) that implements the `--no-git` argument so it doesn't try to find whether there's an update of itself. I've reverted the change.
@bsipocz - am I wrong?

Reply:
afaik you need all your dependencies being available on conda/conda-forge, pip install is not an option (the helpers is not a dependency though as the package includes it as a submodule).

Reply:
It seems that `astropy-helpers` is a dependency, looking at the circle output.

Reply:
I can't see any helpers related output in the circle log, and I don't have time to look into the particulars. In the usual setup `astropy-helpers` should be a submodule and definitely not an installable dependency. Maybe @Cadair can join in to clear up the misunderstandings.

Reply:
In the travis log https://travis-ci.org/conda-forge/staged-recipes/builds/345783282#L339

Reply:
As far as the pypi tarball / conda is concerned astropy-helpers should be a bunch of static files. The extra arguments to setup.py stop it running any of the code to update those files from git or pypi.

I don't know the fine details of using pip though, I wonder if @mwcraig could enlighten us all?

Reply:
The issue is that the first thing that `astropy_helpers` does when it is run is try to update itself...so it isn't a dependency, per se, but it does try to download during the install, which conda catches in the same way it would catch an attempt to download a dependency.

The `--offline` argument to `setup.py` turns off that behavior. I don't know if such an option is available when installing via pip.

So long story short, I'd stick with `setup.py` for now with this package and others that use `astropy_helpers`.